app/templates/suppliers/edit_what_buyers_will_see.html

```diff
@@ -0,0 +1,106 @@
+{% extends "_base_page.html" %}
+{% import "toolkit/forms/macros/forms.html" as forms %}
+
+{% block page_title %}What buyers will see – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace"
+      },
+      {
+        "link": url_for(".dashboard"),
+        "label": "Your account"
+      },
+      {
+        "link": url_for(".supplier_details"),
+        "label": "Company details"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+{% if error %}
```

Comment:
If we don't handle APIErrors we can get rid of this block.

Reply:
ok 👍